lib/utils/strict-binding-parser.html

```diff
@@ -0,0 +1,405 @@
+<!--
```

Comment:
Let's move this file to `lib/mixins`since it's.... a mixin.

Reply:
Done